game-core/src/main/java/games/strategy/engine/lobby/moderator/toolbox/tabs/EventLogTabModel.java

```diff
@@ -22,10 +22,10 @@
 @RequiredArgsConstructor
 class EventLogTabModel {
   @VisibleForTesting
-  static final int PAGE_SIZE = 10;
+  static final int PAGE_SIZE = 30;
```

Comment:
👍 For pagination, even though I'd have hoped there was some API for this in the standard lib for the client side

Reply:
Interesting comment. The paging needs to make its way through to DB, just as the client does not want to get all table data, the server should not read all data either. Without a stateful server, I have trouble seeing how it could be standardized.

Reply:
I was more thinking about the UI support of such a concept.
Having coded a lot of react the last year, I noticed there are a lot of small things like support for this in specific libraries that make your life easier.